src/schemata/definitions/workflow-template/action-objects/webhook.yml

```diff
@@ -90,24 +90,24 @@ additionalProperties:
         - |
           reference to non-field information:
           {
-            "stepPointer": {{workflow_step.pointer}}
+            "step-pointer": {{workflow_step.pointer}}
           }
           The example above will generate a request with the following body:
           {
-            "stepPointer": "/steps/externalStep"
+            "stepPointer": "/steps/external-step"
```

Comment:
Both of these changes are not needed, no? And given that, when writing templates, kebab-case is less common, including other examples in other schema definitions, I suppose it would be more consistent to revert this.